js/ios/Podfile

```diff
@@ -9,7 +9,7 @@ target 'Berty' do
   use_react_native!(:path => config["reactNativePath"])
 
   # override OpenSSL-Universal with own openssl static lib
-  pod 'OpenSSL-Universal', :path => "./OpenSSL-Universal-Override.podspec"
+  # pod 'OpenSSL-Universal', :path => "./OpenSSL-Universal-Override.podspec"
```

Comment:
Please remove both this comment and the podspec

Reply:
I made a clean PR to add libtor back then, so we will only have to check it if we want to reintegrate it. ;)